source/code-snippets/typescript/extend-document.ts

```diff
@@ -0,0 +1,7 @@
+interface YourInterface {
```

Comment:
Disregard if this is consistent with other docs but I think an example would work better here:

```typescript
interface Pet {
  name: string;
  age: number;
  cute: true;
}

const database = client.db("<your database>");
const collection = database.collection<Pet>("<your collection>");
```

Using sentences in the strings as types does work, but the example to a potential newcomer to typescript feels confusing. Again no worries if that's not what we're going for in this particular example.

Reply:
Your edit is not inconsistent with other docs at all. If what I currently have is something that will confuse a TypeScript reader than best not to do it. Thanks for the note!